hooks/pre-commit

```diff
@@ -12,6 +12,26 @@
 #
 # This script does not handle file names that contain spaces.
 
+govet=$(go vet ./lib)
+if [[ ! -z "$govet" ]]; then
```

Comment:
Just a note: `/bin/sh` doesn't recognize `[[`. Instead, you would want to use `[` here, which [is itself a command](https://man.openbsd.org/test.1).